dbump_postgres/postgres.go

```diff
@@ -0,0 +1,87 @@
+package dbump_postgres
```

Comment:
WDYT `dbump_pg` ? :)

Reply:
Nah, let's leave it as it's.

Reply:
I renamed it, `dbump_pg` looks better :)